### PR TITLE
Update .mailmap/AUTHORS. Welcome to SymPy!

### PR DESCRIPTION
**Pull request by oscarbenjamin**
#### Brief description of what is fixed or changed

Update .mailmap and AUTHORS.

#### Release Notes

NO ENTRY

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -983,4 +984,23 @@ Elias Basler <e.e.basler@protonmail.com>
 Brandon David <brandon.david@zoho.com>
 Abhay_Dhiman <abhaysdhimans@gmail.com>
 Tasha Kim <jae_young_kim@brown.edu>
-Ayush-Malik <ayushmalik779@gmail.com>
+Ayush Malik <ayushmalik779@gmail.com>
+Devesh Sawant <devesh47cool@gmail.com>
+Wolfgang Stöcher <wolfgang@stoecher.com>
+Sudeep Sidhu <sudeepmanilsidhu@gmail.com>
+foice <foice.news@gmail.com>
```
Comment (oscarbenjamin) [inline AUTHORS:991]: I'm waiting to hear back whether foice would like to record a different name

File: AUTHORS [hunk 2]
```diff
@@ -983,4 +984,23 @@ Elias Basler <e.e.basler@protonmail.com>
...
+Sayandip Halder <sayandiph4@gmail.com>
+Shubham Agrawal <shubham.ag6845@gmail.com>
+numbermaniac <5206120+numbermaniac@users.noreply.github.com>
+Sakirul Alam <binarysakir@gmail.com>
+Mohammed Bilal <r.mohammedbilal@gmail.com>
+Chris du Plessis <christopherjonduplessis@gmail.com>
+Rishabh Goel <rgoel1999@gmail.com>
+Ansh Mishra <anshmishra471@gmail.com>
+Alex Malins <github@alexmalins.com>
+lcontento <lcontento@users.noreply.github.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1006]: I'm waiting to hear back whether lcontento would like to record a different name

File: AUTHORS [hunk 3]
```diff
@@ -983,4 +984,23 @@ Elias Basler <e.e.basler@protonmail.com>
...
+Sudeep Sidhu <sudeepmanilsidhu@gmail.com>
+foice <foice.news@gmail.com>
+Ben Payne <ben.is.located@gmail.com>
+Muskan Kumar <31043527+muskanvk@users.noreply.github.com>
+noam simcha finkelstein <noam.finkelstein@protonmail.com>
+Garrett Folbe <gmfolbe@yahoo.com>
+Islam Mansour <is3mansour@gmail.com>
+Sayandip Halder <sayandiph4@gmail.com>
+Shubham Agrawal <shubham.ag6845@gmail.com>
+numbermaniac <5206120+numbermaniac@users.noreply.github.com>
```
Comment (oscarbenjamin) [inline AUTHORS:999]: numbermaniac wants to be recorded as numbermaniac: https://github.com/sympy/sympy/pull/20066#issuecomment-703041963

File: AUTHORS (hunk 2 above)
Comment (Coder-RG) [reply AUTHORS:1006]: Is it possible to change my name from Rishabh Goel to Coder-RG? This is also my username so I'd rather use this.

Comment (oscarbenjamin) [reply AUTHORS:1006]: Yes, it is possible. The commit with "Rishabh Goel" can not be removed from the commit history:
```
$ git log | grep 'Rishabh Goel'
Author: Rishabh Goel <rgoel1999@gmail.com>
```
If you would prefer to be recorded as "Coder-RG" in the AUTHORS file then I can add it to the .mailmap file.

Comment (oscarbenjamin) [reply AUTHORS:1006]: I've changed it to Coder-RG

Comment (Coder-RG) [reply AUTHORS:1006]: Thank you! I have made changes in my git setting to reflect the same in future commits.

**Pull request merged**